src/util/promise_utils.js

```diff
@@ -266,13 +266,13 @@ function fork(command, input_args, opts, ignore_rc) {
 function exec(command, options) {
     const ignore_rc = (options && options.ignore_rc) || false;
     const return_stdout = (options && options.return_stdout) || false;
-    const timeout = (options && options.timeout) || 0;
+    const timeout_ms = (options && options.timeout) || 0;
```

Comment:
Did I miss calling sites changes? 
server_diagnostics & os_utils

Reply:
The issue is that I added a util called timeout and the linter does not like it. The interface stayed the same it just a local variable